Emby.Server.Implementations/ScheduledTasks/Tasks/DeleteTranscodeFileTask.cs

```diff
@@ -138,17 +144,17 @@ private void DeleteFile(string path)
             }
         }
 
-        public string Name => "Transcoding temp cleanup";
+        public string Name => "Transcode file cleanup";
```

Comment:
```suggestion
        public string Name => "Transcoded temporary files cleanup";
```

Reply:
This was a deliberate choice on my part to change all strings to "transcode files" rather than "temporary transcode files" since it's less of a mouthful.